src/cljs/netrunner/gameboard.cljs

```diff
@@ -512,7 +512,12 @@
           current-strength [:div.darkbg.strength current-strength])
         (when-let [{:keys [char color]} icon] [:div.darkbg.icon {:class color} char])
         (when server-target [:div.darkbg.server-target server-target])
-        (when subtype-target [:div.darkbg.subtype-target subtype-target])
+        (when subtype-target
+          (let [colour-type (case subtype-target
+                              ("Barrier" "Sentry") (lower-case subtype-target)
+                              "Code Gate" "code-gate"
+                              nil)]
```

Comment:
Feel free to get rid of it if you want. I wanted to see if a PR to your branch worked and this happened instead 😕 😄